classes/04class/exercises/c04-iac01/alanlima/_backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
```

Comment:
Nice use of Work-spaces and Remote site!

Reply:
Yeah, it is a nice way to store the state and I like how it is organised there.
And the best thing.. **it is free** 🤗🤗🤗🤗🎉🎉🎉